scripts/azureml-assets/azureml/assets/release/model_publish_utils/utils.py

```diff
@@ -67,15 +67,27 @@ def _download_git_model(self) -> bool:
             if commit_exists != 0:
                 # TODO: Error handling incase of incorrect commit hash.
                 logger.log_warning("Commit hash doesn't exist. Using model from latest HEAD.")
-        git_path = os.path.join(self.model_dir, '.git')
+        git_path = os.path.join(self.model_dir, ".git")
         shutil.rmtree(git_path, onerror=_onerror)
         return True
 
+    def _download_blobstore_artifacts(self) -> bool:
```

Comment:
Consider using `_download_azure_artifacts` instead of `_download_blobstore_artifacts`.